src/latedeliveryandpenalty/logic/logic.ergo

```diff
@@ -33,8 +33,8 @@ contract LateDeliveryAndPenalty over LateDeliveryAndPenaltyContract {
     ;
 
     // Calculate the time difference between current date and agreed upon date
-    let diff = diffDurationAs(now,agreed,"days");
-    let diffRatio = divideDuration(diff,durationAs(contract.penaltyDuration,"days"));
+    let diff = diffDurationAs(now(),agreed,~org.accordproject.time.TemporalUnit.days);
```

Comment:
Having to full qualify this is a little awkward. Let's chat about a way to make this simpler in a future release.

Perhaps with an alias?
```
import org.accordproject.time.* as Time
```